runelite-client/src/main/java/net/runelite/client/plugins/farmingtracker/FarmingTrackerPanel.java

```diff
@@ -185,6 +187,20 @@ public Dimension getPreferredSize()
 			resize.run();
 
 			materialTab.setOnSelectEvent(() -> config.setPatch(tab));
+			materialTab.addMouseListener(new MouseAdapter()
+			{
+				@Override
+				public void mouseEntered(MouseEvent e)
+				{
+					materialTab.setBackground(ColorScheme.DARKER_GRAY_COLOR.brighter().brighter());
```

Comment:
DARKER_GRAY_COLOR.brighter().brighter() should be close to MEDIUM_GRAY_COLOR right? Can you try that out? should be better performance wise.

Reply:
DARKER_GRAY_COLOR.brighter().brighter() has r=60,g=60,b=60
MEDIUM_GRAY_COLOR is r=77,g=77,b=77

Trying it out, it does seem a little too bright. Should I add a new color to the color scheme? DARKER_GRAY_HOVER_COLOR?
my preview tool stopped working so ill try to fix it and post something

Reply:
Yea I guess it would probably better to add a new color, thought the difference between the two would be smaller.